torchx/lib/torchx.ex

```diff
@@ -17,4 +19,71 @@ defmodule Torchx do
   """
   def device_count(:cuda), do: NIF.cuda_device_count()
   def device_count(_), do: raise("Only CUDA devices can be counted for now.")
+
+  # LibTorch API bindings
+
+  ## Creation
+
+  def arange(from, to, step \\ 1, opts \\ []) do
+    type = opts[:type] || Nx.Type.infer([from, to])
```

Comment:
I think the API in this module should work directly with torch types instead of Nx types. WDYT?